woocommerce/woocommerce
# Move hook-alternatives doc back to GitHub

### Changes proposed in this Pull Request:

Update the document at https://developer.woocommerce.com/docs/cart-and-checkout-hook-alternatives/ to link to a version hosted on GitHub since emojis are not supported on the docs site.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. View the _moved_ document in GitHub (https://github.com/woocommerce/woocommerce/blob/1ef2d2d38a75be5186235677ec02d991accaad49/plugins/woocommerce-blocks/docs/third-party-developers/extensibility/hooks/hook-alternatives.md)
2.  The link in the old doc (https://github.com/woocommerce/woocommerce/blob/1ef2d2d38a75be5186235677ec02d991accaad49/docs/cart-and-checkout-blocks/hooks/hook-alternatives.md) will not work as the new file has not been pushed to trunk yet, however you can compare the link URL to the one above and replace `tree/trunk` with `tree/fix/hook-alternatives-doc` and it should load.




### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
